lib/compiler_options.cmake

```diff
@@ -22,6 +22,7 @@ function(set_compile_options target)
     target_compile_options(${target} PRIVATE -Wconversion -Wsign-conversion)
     target_compile_options(${target} PRIVATE -Wlogical-op -Wdouble-promotion -Wformat=2)
     target_compile_options(${target} PUBLIC $<$<COMPILE_LANGUAGE:CXX>: -fno-rtti>)
+	target_compile_definitions(${target} PUBLIC HAS_UNCAUGHT_EXCEPTIONS)
```

Comment:
This change was only necessary when using date.h.
It can be removed.

Reply:
Should I delete this file?

Reply:
Well, not the file but please go ahead and delete the file.